lib/main.go

```diff
@@ -4,7 +4,7 @@ import "github.com/status-im/status-go/api"
 
 var statusBackend = api.NewStatusBackend()
 
-// Technically this package supposed to be a lib for
+// main; Technically this package supposed to be a lib for
```

Comment:
I don't know this one. Can you please point me to a fragment in Effective Go that explains this?